src/nfa/operations.cc

```diff
@@ -789,6 +819,7 @@ Nfa Mata::Nfa::reduce(const Nfa &aut, bool trim_input, StateToStateMap *state_ma
     return result;
 }
 
+//Might be a method, but it will not be in-place anyway, so I don't know.
```

Comment:
We agreed that functions should return new instance and methods should compute in-place, if at all possible. Therefore, this function should remain a function.

Reply:
ok

Reply:
I agree.